File: 프로그래머스/lv1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        a = []
        for j in range(commands[i][0] - 1, commands[i][1]):
            a.append(array[j])
        answer.append(sorted(a)[commands[i][2] - 1])
    return answer

File: 프로그래머스/lv1/42748. K번째수/K번째수.kt
class Solution {
    fun solution(array: IntArray, commands: Array<IntArray>): IntArray {
        val answer = IntArray(commands.size)
        for ((i, command) in commands.withIndex()) {
            val s = array.slice(command[0] - 1 until command[1])
            answer[i] = s.sorted()[command[2] - 1]
        }
        return answer
    }
}